CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.5.3
```

Comment:
You can leave this as `# Unreleased`. We usually bump the gem in a separate, subsequent PR. We usually do this to prevent immediate publishing of the gem, in case there's some integration testing necessary.